app/ante_no_seq.go

```diff
@@ -64,7 +76,13 @@ func NewAnteHandler(
 		ante.NewValidateSigCountDecorator(ak),
 		ante.NewSigGasConsumeDecorator(ak, sigGasConsumer),
 		ante.NewSigVerificationDecorator(ak, signModeHandler),
-		// ante.NewIncrementSequenceDecorator(ak),
+		ante.NewIncrementSequenceDecorator(ak),
```

Comment:
@czarcas7ic e2e tests pass with the sequence here, can we add this back in?

Reply:
No, I think this should remain removed

Reply:
Added back